.golangci.next.reference.yml

```
@@ -1540,6 +1540,14 @@ linters-settings:
     patterns:
       - ".*"
 
+  recv:
+    # Maximum length for a receiver name.
+    # Default: 3
+    max-name-length: 6
+    # Allow to use pointer receiver and non pointer receiver on the same struct.
```

This would be better I think

```suggestion
    # Allow using pointer receiver and non pointer receiver on the same struct.
```

---

also you may consider to use 

```suggestion
    # Allow using pointer receiver and non-pointer receiver on the same struct.
```

if you agree with this PR https://github.com/ldez/recv/pull/1